csharp/ql/integration-tests/all-platforms/dotnet_run/test.py

```diff
@@ -0,0 +1,17 @@
+from create_database_utils import *
+
+# no arguments
+run_codeql_database_create(['dotnet run'], test_db="test-db", db=None, lang="csharp")
+
+# no arguments, but `--`
+run_codeql_database_create(['dotnet clean', 'rm -rf test-db obj bin', 'dotnet run --'], test_db="test2-db", db=None, lang="csharp")
```

Comment:
It is possible to delete at least the 'obj' and 'bin' folder as a part of the clean target (if the following is added to the project folder).
```csharp
<Target Name="DeleteBinObjFolders" BeforeTargets="Clean">
   <RemoveDir Directories=".\bin" />
   <RemoveDir Directories=".\obj" />
 </Target>
```

Was it the `test-db` folder that also causes failures when running multiple tests (I suppose it is because the source code is included in the testdb folder)?

Reply:
Yes. Can we fix that follow-up?